code/drasil-code/lib/Language/Drasil/Code/Imperative/Descriptions.hs

```diff
@@ -214,4 +216,4 @@ physAndSfwrCons = do
         (inputs $ codeSpec g)
   return $ stringList [
     if not (any isPhysC cns) then "" else "physical constraints",
-    if not (any isSfwrC cns) then "" else "software constraints"]
\ No newline at end of file
+    if not (any isSfwrC cns) then "" else "software constraints"]
```

Comment:
A minor thing, but it is convention to end files with a trailing newline.
```suggestion
    if not (any isSfwrC cns) then "" else "software constraints"]
    
```

Reply:
It seems like there was already a trailing newline there and you added an additional one? Now there are two trailing newlines.

Reply:
Hmm... my diff looks like there was a trailing line removed - I'll see what it looks like in the branch

Reply:
Yes, you are right - my bad!